M2/Macaulay2/packages/CellularResolutions/tests.m2

```diff
@@ -27,10 +27,10 @@ assert(l1=!=l2);
 assert(dim l1==1);
 assert(dim l2==1);
 C = cellComplex(QQ,{l1,l2});
-CchainComplex = chainComplex C;
```

Comment:
Minor point: I think a some of the changed lines here aren't necessary since `CchainComplex` is just a variable, and since `C` is a (cell) complex having "chain" in the variable name doesn't hurt.